lib/vtls/sectransp.c

```diff
@@ -2599,8 +2599,9 @@ sectransp_connect_step2(struct Curl_easy *data, struct connectdata *conn,
 #if CURL_BUILD_MAC_10_6
       /* Only returned when kSSLSessionOptionBreakOnCertRequested is set */
       case errSSLClientCertRequested:
-        failf(data, "The server has requested a client certificate");
-        break;
+        failf(data, "Server requested a client certificate during the SSL "
+              "handshake");
+        return  CURLE_SSL_CLIENTCERT;
```

Comment:
Can we make this say just "during the handshake"? While I don't really mind SSL being used interchangeably with TLS when used generically, "SSL handshake" is a specific part of a protocol interaction which this error doesn't actually deal with.

Reply:
Fixed.